bodhi/tests/server/tasks/test_updates.py

```diff
@@ -304,6 +306,91 @@ def test_update_not_found(self, work_on_bugs, fetch_test_cases, sleep):
         self.assertEqual(fetch_test_cases.call_count, 0)
         sleep.assert_called_once_with(1)
 
+    @mock.patch('bodhi.server.tasks.updates.UpdatesHandler.fetch_test_cases')
+    @mock.patch('bodhi.server.tasks.updates.UpdatesHandler.work_on_bugs')
+    @mock.patch.dict(config.config, [('test_gating.required', True)])
+    def test_rawhide_update_edit_move_to_testing(self, work_on_bugs, fetch_test_cases, sleep):
+        """
+        Assert that a pending rawhide update that was edited gets moved to testing if all the
+        updates builds are signed.
```

Comment:
I'd phrase it "...if all the **builds in the update** are signed.", likewise in the other test methods.